ff_net_worth_calculator/__init__.py

```diff
@@ -1,37 +1,71 @@
 import json
 import os
 import requests
+import sys
 
 _ROOT = os.path.abspath(os.path.dirname(__file__))
+_timeout = 0.1
```

Comment:
IMHO this timeout is set too small considering it is not hard to find crappy map servers which nearly take longer for connection establishment.

Reply:
What would be a better option? `0.3`?

Reply:
I would go for a 1 second timeout, possibly adding the ability of setting a custom timeout via an optional function argument.

Reply:
That would mean that the loading of the files would take even longer